src/main/java/com/josketres/rfcfacil/JuristicPersonTenDigitsCodeCalculator.java

```diff
@@ -65,7 +66,7 @@ public String calculate() {
                 .flatMap(this::expandRomanNumerals)
                 .toArray(String[]::new);
 
-        return threeDigitsCode() + "-" + birthdayCode();
+        return " " + threeDigitsCode() + birthdayCode();
```

Comment:
En las reglas de la conformación dice que se debe agregar el espacio, ya
que al mapear los caracteres, si no se encuentra el espacio, la homoclave
no se genera correctamente, hice los ejercicios correspondientes para
validarlo.

ISC. Mario Jiménez Carrasco.
Subdirección Técnico y de Negocios
Desarrollo de Software.
Gobierna SCP.

On Sun, Jul 3, 2016 at 3:00 AM, Josue Zarzosa <notifications@github.com>
wrote:

> In
> src/main/java/com/josketres/rfcfacil/JuristicPersonTenDigitsCodeCalculator.java
> <https://github.com/josketres/rfc-facil/pull/7#discussion_r69387288>:
>
> > @@ -65,7 +66,7 @@ public String calculate() {
> >                  .flatMap(this::expandRomanNumerals)
> >                  .toArray(String[]::new);
> >
> > -        return threeDigitsCode() + "-" + birthdayCode();
> > +        return " " + threeDigitsCode() + birthdayCode();
>
> ¿Para qué le concatenamos un espacio aquí, si comoquiera lo vamos a borrar
> más tarde tenDigitsCode.trim()?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/josketres/rfc-facil/pull/7/files/a9d20a086440e8b43ac7f0bf06e70cbef3580325#r69387288>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE4qFRK2Co0cJf1s_1tgC9b0XZKDora4ks5qR2w0gaJpZM4JDXvs>
> .
>